SurveyApp/Modules/Splash/SplashViewController.swift

```diff
@@ -83,8 +84,15 @@ extension SplashViewController {
     }
 
     private func beginAnimation() {
-        UIView.animate(withDuration: 1.0, delay: 0.5) { [weak self] in
-            self?.logoImageView.alpha = 1.0
-        }
+        UIView.animate(
+            withDuration: 1.0,
+            delay: 0.5,
+            animations: { [weak self] in
+                self?.logoImageView.alpha = 1.0
+            },
+            completion: { [weak output] _ in
+                output?.animationDidFinish()
+            }
+        )
```

Comment:
No need weak reference here. 

Also what happen if you start the app and quickly swipe to home and come back?

Reply:
@blyscuit Sorry, I'm confused. Could you help clarify? Do you mean open the app when logged in and open again?

Reply:
No worry, it's all good.

Reply:
Oh okay 🙆‍♀️